falco.yaml

```diff
@@ -492,6 +492,7 @@ webserver:
   # the appropriate number of threads based on the number of online cores in the system.
   threadiness: 0
   listen_port: 8765
+  listen_address: 0.0.0.0
```

Comment:
Should we explicitly mention that only TCP/IPv4 is supported? Compare to grpc bind_address implementation (which btw perhaps could be cleaned up as well by someone in the future).

Reply:
Do you mean TCP/IP VS unix socket ?

Reply:
And Ipv4 only.

Reply:
On the TCP side, shouldn't IPV6 also work with httplib ?

Reply:
You wanna try? If we easily can, we should. Perhaps we just need the ipv6 regex parser then?

Reply:
This LGTM; re ipv6, i think it is just a mattter of updating the regex (and tests) as Melissa said.
We can do that in a follow up PR anyway!

Reply:
Sorry for the late reply, I did a small test to see if the webserver starts when listening to an IPV6 address other than `::` (which is working and thus already a good sign) and it does for pure IPV6 addresses. From the looks of it, literal IPV6 addresses are not supported. I currently don't know whether IPV4 mapped address work either.

Related to that topic, we might want to discuss things a bit further with regard to the IPV6 support design. Do we just reuse the same setting but with a regex that can validate both IPV4 and IPV6 values ? Or should there be a separation between both through the use of `listen_address` and `listen_ip6_address` ?

Reply:
> Do we just reuse the same setting but with a regex that can validate both IPV4 and IPV6 values ?

I would say yes. I would reuse the same config allowing users to specify either an ipv4 or an ipv6